### PR TITLE
Add ID to Export

### PR DESCRIPTION
**Pull request by skipman**
Makes the ID of the detailed data records selectable for export

### CODE REVIEW

File: library/Leads/Export.php [hunk 1]
```diff
@@ -263,10 +277,14 @@ protected function getExportData($objConfig, $arrIds=null)
             if ($objConfig->export == 'all') {
                 \System::loadLanguageFile('tl_lead_export');
 
+                $arrHeader[] = $GLOBALS['TL_LANG']['tl_lead_export']['_pid']; // insert ID in export-file (added by CO/BODA - Ronald Boda)
```
Comment (aschempp) [inline library/Leads/Export.php:280]: this translation does not seem to exist?

**Review by aschempp — CHANGES_REQUESTED** [review]
Thanks! This looks good in general, but please remove all the comments. We have GIT (blame) for references who added what and where 😊

File: library/Leads/Export.php (hunk 1 above)
Comment (Skipman) [reply library/Leads/Export.php]: pushed changed files and additional translation files

**Comment by Skipman** [comment]
As requested combined PR #110 and PR #109 into #109 only

**Comment by aschempp** [comment]
I have no implemented this in the upcoming version 3 (the field is called `_id`, see 39983f4809f8e09b6abb39be4a43a03b199c893c).

**Pull request closed without merging**